### ISSUE TITLE
Cannot use mbknor-jackson-jsonschema with JDK 11

### ISSUE THREAD

**Issue by jflefebvre06** [mbknor/mbknor-jackson-jsonSchema#101]
`requires mbknor.jackson.jsonschema.2.12;` is not valid

Could you please add module name to meta-inf

**Comment by mbknor** [comment]
I'm not sure what the the problem actually is. Please provide more info or PR with proposed changes

**Comment by jflefebvre06** [comment]
When working with JDK9 or highter and import non jdk9 artifacts, Jigsaw determine the module name of the jar like this :
* look into `META-INF/MANIFEST.MF` the `Automatic-Module-Name` (http://branchandbound.net/blog/java/2017/12/automatic-module-name/)
* if not provided determin the module name based on jar file name

In our case, the jar name is `mbknor-jackson-jsonschema_2.12-1.0.34.jar` but the Jigsaw specification says the jar name must not contains '_'

`jar --describe-module --file=mbknor-jackson-jsonschema_2.12-1.0.34.jar`

![image](https://user-images.githubusercontent.com/2685607/69129636-15d42280-0aaf-11ea-97b2-9b5b5fe51808.png)

**Solution:**

Add to your pom.xml

```xml
<plugin>
  <groupId>org.apache.maven.plugins</groupId>
  <artifactId>maven-jar-plugin</artifactId>
  <configuration>
    <archive>
      <manifestEntries>
        <Automatic-Module-Name>mbknor.jackson.jsonschema</Automatic-Module-Name>
      </manifestEntries>
    </archive>
  </configuration>
</plugin>
```

**Comment by mbknor** [comment]
Thanks a lot for the detailed info.. Please have a look at the PR. Do you think this fixes the issue?

**Comment by jflefebvre06** [comment]
Yes your fix will work fine thanks

**Comment by mbknor** [comment]
Thanks for checking.

**Comment by mbknor** [comment]
I've released 1.0.36 and it is on its way to maven central
